docs/iris/example_code/General/polynomial_fit.py

```diff
@@ -0,0 +1,51 @@
+"""
+Fitting a polynomial
+====================
+
+This example demonstrates adding a polynomial fit to a 1D plot of data from
```

Comment:
Actually my primary intent with this example was to demonstrate the fitting process but I like the extension. Also the plot doesn't work anything like as well unless the fit is added to the cube's metadata!